File: iniciante/1041/1041.py
# 1041 - Coordenadas de um Ponto - beecrowd
# Iniciante - Nível 3

x, y = map(float, input().split())

if x == 0 and y == 0: print("Origem")
elif x == 0 or y == 0: print("Eixo Y" if x == 0 else "Eixo X")
else: print(("Q1" if y > 0 else "Q4") if x > 0 else ("Q2" if y > 0 else "Q3"))

File: iniciante/1041/1041.c
// 1041 - Coordenadas de um Ponto - beecrowd
// Iniciante - Nível 3

#include <stdio.h>

int main()
{
    double x, y;
    scanf("%lf %lf", &x, &y);

    if (x == 0 && y == 0) puts("Origem");
    else if (x == 0 || y == 0) printf("%s\n", (x == 0.0) ? "Eixo Y" : "Eixo X");
    else puts(x > 0 ? (y > 0 ? "Q1" : "Q4") : (y > 0 ? "Q2" : "Q3"));

    return 0;
}
